Cloudmarque.Azure/Public/IaaS/vm/Set-CmAzIaasUpdateManagement.ps1

```
@@ -65,13 +68,23 @@ function Set-CmAzIaasUpdateManagement {
 				Write-Error "Frequency not recognised." -Category InvalidArgument -CategoryTargetName "frequency"
 			}
 
+			$currentDate = (Get-Date).date.addDays(1)
+
+			if (!($scheduleSetting.startTime -Is [DateTime]) -or $scheduleSetting.startTime -lt $currentDate) {
+				$scheduleSetting.startTime = $currentDate
```

same thing. As per me, If customer has provided a certain date which is wrong which should fail instead of setting to some default default as most of the time its a typo and the created resource is hard to overwrite.